### ISSUE TITLE
/root/.my.cnf prevents to use other mysql tools

### ISSUE THREAD

**Issue by laurentj** [aalaesar/install_nextcloud#31]
db_mysql.yml creates a /root/.my.cnf, with credentials of the root user of mysql.

Is it really needed ?

This file prevents some tools, like backup-manager (executed with the linux root user), to work correctly, when it uses an other mysql user than root. 

For example, backup manager is using this command :

```
/usr/bin/mysqldump --defaults-extra-file=/root/.backup-manager_my.cnf --opt -ubackup -hlocalhost -P3306 --all-databases
```

Even if the password of the 'backup' user is into /root/.backup-manager_my.cnf,  /root/.my.cnf has priority over this file, and so it take the password stored into /root/.my.cnf. 

/root/.my.cnf should not contain any credentials.

(and I don't want to do backups with a user that have all privileges).

**Comment by aalaesar** [comment]
Hello @laurentj and thank you for submitting this issue.
I apologize for my late answer, I was away.
You have a good point.
The .my.cfg is used by the mysql modules to access root privileges in order to create the nexcloud user and database.
This has been made to use easily most of database managment operation.
I'll look on this point in the next few days.
Regards, Aal.

**Comment by egmont1227** [comment]
This should be fixed with PR #58

**Comment by aalaesar** [comment]
should be fixed by #58
